src/user-guide/task-implementation/run-modes.rst

```diff
@@ -0,0 +1,207 @@
+.. _task-run-modes:
+
+Task Run Modes
+==============
+
+.. versionadded:: 8.4.0
+
+   * Before 8.4.0 run modes could only be set for the workflow, not
+     for individual tasks.
+   * Skip mode.
+
+
+As well as the default task implementation ("live" mode)
+other implementations can be selected:
+
+* By setting the scheduler run mode as an argument to the
+  ``cylc play --mode`` command ("simulation" and "dummy" modes).
+* By setting :cylc:conf:`[runtime][<namespace>]run mode` for
+  "skip" mode.
+
+#. :ref:`task-run-modes.simulation`: Intended to be used
+   when developing a workflow, the scheduler simulates
+   running a task without creating a "real" job.
+#. :ref:`task-run-modes.dummy`: Intended to be used
+   when developing a workflow, the scheduler replaces
+   the workflow scripts with dummy scripts and returns
+   to the default ("live") submission pathway.
+#. :ref:`task-run-modes.skip`: Intended to allow more control of the workflow,
+   the scheduler marks task outputs as completed.
+
+.. note::
+
+   The simulation and dummy workflow run modes ignore the
+   :cylc:conf:`[runtime][<namespace>]run mode` setting.
+
+
+.. _task-run-modes.dummy:
+
+Dummy Mode
```

Comment:
I don't think dummy and simulation should have been moved to a "task run mode" page?

Reply:
Arguable, because they are still task modes, but I take your point, and will move them back.